client/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 // import { Button, AppBar, Box, Toolbar, IconButton, Grid, Typography } from '@mui/material';
```

Comment:
Remove commented out code for general code hygiene.

Reply:
Ok!